SUPPORT.md

```diff
@@ -1,5 +0,0 @@
-# Support
```

Comment:
my only hesitation here is removing this file entirely.  This came from the github-boilerplate, I believe, which was informed by some standards org's suggested github files.   Would we remove this from the boilerplate and all repos?

Reply:
oh yeah, shouldn't this just be updated with the link to the simularium forum?

Reply:
I moved the contents of this file to README.md because I thought this info didn't warrant an entire separate file, and I thought README.md should have that info anyway. But I don't mind if we keep this file for completeness' sake if that's what we want.

Reply:
I guess I would feel fine with removing it in a project that thought it wasn't necessary but I would leave it in the github-boilerplate

Reply:
Okay, then I will leave this PR as is :) And I won't do anything with the boilerplate template.